src/Stratis.Bitcoin/Configuration/Settings/ConnectionManagerSettings.cs

```diff
@@ -29,8 +40,14 @@ public ConnectionManagerSettings()
         /// Loads the ConnectionManager related settings from the application configuration.
         /// </summary>
         /// <param name="nodeSettings">Application configuration.</param>
-        public void Load(NodeSettings nodeSettings)
+        public ConnectionManagerSettings Load(NodeSettings nodeSettings)
```

Comment:
Please move Load body to the constructor

Reply:
Sure. We can add that to this PR.